src/hyperlane/HyperlaneHelper.sol

```diff
@@ -11,15 +11,25 @@ interface IMessageRecipient {
 }
 
 contract HyperlaneHelper is Test {
-    function help(address mailbox, uint256 forkId, Vm.Log[] calldata logs) external {
-        _help(mailbox, 0x769f711d20c679153d382254f59892613b58a97cc876b249134ac25c80f9c814, forkId, logs);
+    function help(address mailbox, uint32 originDomain, uint256 forkId, Vm.Log[] calldata logs) external {
```

Comment:
Given that the `originDomain` only matters when gas is estimated, I think `originDomain` should be removed from the `help` functions and instead provide a `helpAndEstimate` function.

For the `help` functions, they would still call `_help(address mailbox, uint32 originDomain, ...)` but pass a bogus `originDomain`.

This is for both HL and LZ/SG.

`_help` could include a boolean arg to whether estimate or not.

```js
function helpWithEstimate(....) {
    bool enableEstimates = vm.envOr("ENABLE_ESTIMATES", false);
    _help(...., enableEstimates);
}

Reply:
Sounds good! I was thinking the same as well.